todo-app/src/List-Item.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+
 export default class ListItem extends React.Component {
 
     state = {
```

Comment:
You're still handling state in two places, keeping the checked state inside the `ListItem`. Is it possible to keep this `checked` state inside the todo objects? What if we needed to list the number of completed todos in `Main.js`? See if you can lift this state up.